RELEASE.md

```diff
@@ -1,3 +1,31 @@
+# Release 1.0.0
+
+## Breaking Changes
+
+- New parameter `tensor_fields` will replace `non_tensor_fields` in the `add_documents` endpoint. Only fields in `tensor_fields` will have embeddings generated, thereby offering more granular control as to which fields to vectorise. The `non_tensor_fields` parameter is deprecated and will be removed in a future release. Calls to `add_documents` with neither of these parameters specified will now fail [#538](https://github.com/marqo-ai/marqo/pull/538).
+
+- Single KNN Vector Optimisation. Only one OpenSearch KNN vector field will be created per index, instead of one per field. This will result in a single HNSW graph, thus increasing search speed by eliminating the need to load multiple graphs in and out of memory. Note that indexed documents will now have a different OpenSearch document structure, so documents indexed with previous Marqo versions cannot be searched with this version, and vice versa [#530](https://github.com/marqo-ai/marqo/pull/530).
```

Comment:
Reduce implementation detail explanation. This sentence can be removed: `This will result in a single HNSW graph, thus increasing search speed by eliminating the need to load multiple graphs in and out of memory`

Reply:
done